### ISSUE TITLE
When I adjust the range of the point_cloud_range parameter to a rectangle, for example point_cloud_range = [-60.0, -54.0, -5.0, 60.0, 54.0, 3.0], there may be some problems here?

### ISSUE THREAD

**Issue by huaaaa1** [NVlabs/FocalFormer3D#23]
After adjusting the point cloud range, I adjusted some parameters , such as 'Sparse_shape','post_center_range','grid_size', and I fixed the shape of 2D_grid generated by focal_decoder.py in lines 537 and 538 to make it consistent with the shape of BEV Feature. However, I find that after such modification, the loss of the model in the training process gradually drops to a certain value and becomes stable, but the matched_ious of the training set is always poor which is about 0.45(matched_ious can get up to 0.8-0.9 with Square range ). Therefore, I would like to ask whether the model can support the change of the point cloud range to a rectangle, and if so, what should I modify? Looking forward to your reply. Thank you very much.

**Comment by chenyilun95** [comment]
I am not sure if there some potential bugs to support the rectangle shape either in data loader or model. If the matched_iou of range [-60.0, -60.0, -5.0, 60.0, 60.0, 3.0] is correct while [-60.0, -54.0, -5.0, 60.0, 54.0, 3.0] is wrong, it looks like the x/y dimension order of create_2d_grid is reversed as they will be flattened in the following attention modules.
